apps/client/src/app/components/account-detail-dialog/account-detail-dialog.component.ts

```diff
@@ -140,6 +140,31 @@ export class AccountDetailDialog implements OnDestroy, OnInit {
     this.dialogRef.close();
   }
 
+  public onAddAccountBalance({
+    date,
+    balance
+  }: {
+    date: string;
+    balance: number;
+  }) {
+    const formattedDate = new Date(date);
+    this.dataService
+      .postAccountBalance({
+        userId: this.user.id,
```

Comment:
Remove from client

Reply:
I am not sure what you mean here?

Reply:
Yes, exactly. If you get it from the client, you need to verify it first.